openshift/templates/issuer-agent/issuer-agent-deploy.a2a.param

```diff
@@ -3,39 +3,40 @@
 # Component: .
 # Template File: ../openshift/templates/issuer-agent/issuer-agent-deploy.yaml
 #=========================================================
-NAME=agent
-APP_NAME=issuer
+NAME=agent-a2a
+APP_NAME=issuer-a2a
 APP_GROUP=a2a
-SUFFIX=-a2a
+SUFFIX=
```

Comment:
the suffix was removed from the a2a profile because the template attaches the suffic to the wallet host name. In the case of a2a, it uses a shared wallet pod `issuer-kit-wallet` and then add the suffix on `issuer-kit-wallet-a2a`, which doesn't exist